### DISCUSSION TITLE
群晖docker natfrp ssl问题

### DISCUSSION THREAD

**Discussion by ghost** [natfrp/wiki#243] [使用问题]
开启了ssl功能，然后群晖docker里面怎么设置才能开启ssl，用自己的证书

natfrp的默认证书在哪里目录呢？需要如何映射和更改默认证书替换为自己的证书。
![image](https://user-images.githubusercontent.com/42706238/208284880-a0f76843-725c-4709-ab0a-2ddd14d75200.png)

![image](https://user-images.githubusercontent.com/42706238/208284835-a57e660e-b43e-4281-9954-ef929c55e5c7.png)

还是说群里里面替换群晖证书就可以？

![image](https://user-images.githubusercontent.com/42706238/208284861-3ca5ff4e-0a2f-4254-aea8-34ec6767d88e.png)

**Comment by fakeboboliu** [comment]
如果已按教程进行的话，替换群晖证书即可
